hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -170,10 +168,12 @@ public static long totalNewMessages(OffsetRange[] ranges) {
   /**
    * Configs to be passed for this source. All standard Kafka consumer configs are also respected
    */
-  static class Config {
+  public static class Config {
```

Comment:
should we really need set the class to `public` just for testing?

Reply:
@leesf basically I did not want to create 2 copies of the variable DEFAULT_MAX_EVENTS_FROM_KAFKA_SOURCE. Hence made Config class public. If we define the same variable at 2 places (one in above and one in TestKafkaSource), then there are chances that they might go out of sync (someone might update one of them and forget to change the other). WDYT?

Reply:
Resonable.